rita_common/src/dashboard/wifi.rs

```diff
@@ -1,18 +1,18 @@
 //! These endpoints are used to modify mundane wireless settings
 
+use crate::dashboard::nickname::maybe_set_nickname;
+use crate::{RitaCommonError, KI};
```

Comment:
why make all these changes? why not import from rita client?

Reply:
fixed